build.gradle

```diff
@@ -116,9 +116,12 @@ task insertVersion() {
 }
 
 def insertProjectVersion(String project, String abbreviation) {
-	Properties properties = new Properties()
-	file("${project}/version.properties").withInputStream { properties.load(it) }
-	String version = properties.previousVersion
+	final def stdout = new ByteArrayOutputStream()
```

Comment:
i know we are using a lot of command line thingies execution already, but, maybe grgit would be a nice option here - http://ajoberstar.org/grgit/grgit-tag.html 

but it looks fine form my side

Reply:
I'll look into it. Soon™.

Reply:
Looked into it. Implemented it.